Dockerfile

```diff
@@ -51,15 +58,17 @@ USER mirrorbits
 
 COPY config/mirrorbits.conf /etc/mirrorbits/mirrorbits.conf
 
-COPY --from=mirrorbits /mirrorbits/tini /bin/tini
-
-COPY --from=mirrorbits /mirrorbits/mirrorbits /usr/bin/mirrorbits
+COPY --from=build /mirrorbits/tini /bin/tini
+COPY --from=build /mirrorbits/bin/mirrorbits /usr/bin/mirrorbits
+COPY --from=build /mirrorbits/templates /usr/share/mirrorbits/templates
 
-COPY --from=mirrorbits /mirrorbits/templates /usr/share/mirrorbits/templates
+LABEL \
+  io.jenkins-infra.tools="mirrorbits,tini" \
+  io.jenkins-infra.tools.mirrorbits.version="${mirrorbits_version}" \
+  io.jenkins-infra.tools.tini.version="${tini_version}" \
+  repository="https://github.com/jenkins-infra/docker-mirrorbits"
 
-LABEL io.jenkins-infra.tools="mirrorbits,tini"
-LABEL io.jenkins-infra.tools.mirrorbits.version="${mirrorbits_version}"
-LABEL io.jenkins-infra.tools.tini.version="${tini_version}"
+EXPOSE 8080 3390
```

Comment:
3390 is the port to use for the gRPC client. Not used by default (requires configuration in mirrorbits.conf)